src/main/java/core/mate/academy/model/TruckProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.model;
+
+import java.util.List;
+import core.mate.academy.service.MachineProducer;
+
+public class TruckProducer implements MachineProducer<Truck> {
+
+    @Override
+    public List<Truck> get() {
+        Truck volvo = new Truck("Name : Volvo CR900" , "Color : Blue", 2000,
+                "diesel");
+        Truck man = new Truck("Name : Man RY200", "Color : Black", 1900, "diesel");
```

Comment:
Consider removing the 'Name :' and 'Color :' prefixes from the truck names and colors for consistency and clarity. The constructor parameters should directly represent the name and color values without additional text.